File: A1/A1-020/A1-020.cpp
#include<iostream>
using namespace std;

int main()
{
    // This makes std::cin and std::cout faster
    ios_base::sync_with_stdio(false); cin.tie(0);

    // Input
    int a, b, c;
    cin >> a >> b >> c;
    // Output
    if(a < b && b < c) { cout << "increasing"; }
    else if(a > b && b > c) { cout << "decreasing"; }
    else { cout << "neither"; }
    return 0;
}

File: A1/A1-020/A1-020.py
# --------------------------------------------------
# File Name : A1-020.py
# Problem   : Increasing Decreasing
# Author    : Worralop Srichainont
# Date      : 2026-02-08
# --------------------------------------------------

# Input numbers
a = int(input())
b = int(input())
c = int(input())

# Output
if a < b < c:
    print("increasing")
elif a > b > c:
    print("decreasing")
else:
    print("neither")
